marklogic-data-hub/src/main/resources/ml-modules/root/data-hub/5/data-services/models/getPrimaryEntityTypes.sjs

```diff
@@ -1,5 +1,5 @@
 /*
-  Copyright 2012-2019 MarkLogic Corporation
+  Copyright (c) 2020 MarkLogic Corporation
```

Comment:
While this is a good change, let's hold off on in and use a separate JIRA/PR to clean up the copyright years. I did a quick search, and there are a lot more issues to fix than just these. It'll shrink the size of this PR which is good too, since it has some complex stuff in it.

Reply:
Done